storeshield/routes/files.py

```
@@ -0,0 +1,117 @@
+import os
+from asyncio import StreamReader
+
+import aiohttp
```

Ruff (0.8.2)

`aiohttp` imported but unused

Remove unused import: `aiohttp`

(F401)